models/business.js

```diff
@@ -14,10 +14,10 @@ const businessSchema = new mongoose.Schema({
   state: { type: String },
   country: { type: String },
   mainPicture: { type: String },
+  email: { type: String, required: true, unique: true },
   status: { type: Number, default: 1 },
-  email: { type: String },
-  features: [{ type: ObjectId, ref: 'features' }],
-  services: [{ type: ObjectId, ref: 'services' }]
+  features: { type: ObjectId, ref: 'features' },
```

Comment:
@cynthiaeddy -> Hi! Can you please elaborate on why you made this modification?

Reply:
@CodeMeKathy Hi! I don't know why that showed up - here's what I have on my local (screenshot). I'd say delete it.
<img width="851" alt="Screen Shot 2021-11-13 at 9 53 35 AM" src="https://user-images.githubusercontent.com/15850286/141650420-69b25c7f-6773-4439-ba66-8477298135bb.png">

Reply:
Hi @cynthiaeddy! Your picture aligns with the committed changes; the `businessSchema` features property is no longer set to an array of `ObjectId`s. My initial comment was an attempt to understand better the reason behind the change here and to services. I do not believe I can delete it. You would need to undo/revert this change and commit the modifications to the PR.